ui/perfherder/compare/CompareTable.jsx

```diff
@@ -141,6 +187,15 @@ export default class CompareTable extends React.PureComponent {
                 ) : null}
               </td>
               <td className="text-right">
+                {!hasSubtests && user.isLoggedIn && (
```

Comment:
We should hide the button if we `isAggregated === true` & we don't have data for **New**.